### ISSUE TITLE
faction.id returns faction.name

### ISSUE THREAD

**Issue by qqqbbb** [MWSE/MWSE#221]
```
for k, faction in pairs(tes3.dataHandler.nonDynamicData.factions) do
	mwse.log('%s    %s', faction.id, faction.name);
end
```

**Comment by adituv** [comment]
Confirmed: I created a plugin with a new faction (id = TestFaction_Id, name = TestFaction_Name) and got TestFaction_Name showing as both the output for id and name.

I believe this is probably a bug in the native Morrowind getObjectId function, however, as Faction inherits id from BaseObject, and that just calls getObjectId from the BaseObject vTable.

I've created a possible patch ([diff](https://github.com/MWSE/MWSE/compare/master...adituv:221-faction-id)) but I'm not sure whether it should be applied for the above reason.
